File: app/src/main/java/com/toosmart/teens/MainActivity.java
package com.toosmart.teens;

import androidx.appcompat.app.AppCompatActivity;

import android.content.Intent;
import android.net.Uri;
import android.os.Bundle;
import android.view.View;
import android.widget.Button;

public class MainActivity extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);

        Button button1 = (Button) findViewById(R.id.page1_button1);
        Button button2 = (Button) findViewById(R.id.page1_button2);
        Button button3 = (Button) findViewById(R.id.page1_button3);
        Button button4 = (Button) findViewById(R.id.page1_button4);
        Button button5 = (Button) findViewById(R.id.page1_button5);
        Button button6 = (Button) findViewById(R.id.page1_button6);

        button1.setOnClickListener(new View.OnClickListener() {
            public void onClick(View v) {
                String url = "https://henricoprevention.org/youth-leadership/";
                Intent intent = new Intent(Intent.ACTION_VIEW);
                intent.setData(Uri.parse(url));
                startActivity(intent);
            }
        });

        button2.setOnClickListener(new View.OnClickListener() {
            public void onClick(View v) {
                String url = "https://henricoprevention.org/resources/";
                Intent intent = new Intent(Intent.ACTION_VIEW);
                intent.setData(Uri.parse(url));
                startActivity(intent);
            }
        });
        button3.setOnClickListener(new View.OnClickListener() {
            public void onClick(View v) {
                String url = "https://henricoprevention.org/service-partners/";
                Intent intent = new Intent(Intent.ACTION_VIEW);
                intent.setData(Uri.parse(url));
                startActivity(intent);
            }
        });
        button4.setOnClickListener(new View.OnClickListener() {
            public void onClick(View v) {
                String url = "https://www.instagram.com/henricotoosmart/";
                Intent intent = new Intent(Intent.ACTION_VIEW);
                intent.setData(Uri.parse(url));
                startActivity(intent);
            }
        });
        button5.setOnClickListener(new View.OnClickListener() {
            public void onClick(View v) {
                String url = "https://www.youtube.com/channel/UCPoL8cFoxCdPlZFsa8YoLlQ";
                Intent intent = new Intent(Intent.ACTION_VIEW);
                intent.setData(Uri.parse(url));
                startActivity(intent);
            }
        });
        button6.setOnClickListener(new View.OnClickListener() {
            public void onClick(View v) {
                String url = "";
                Intent intent = new Intent(Intent.ACTION_VIEW);
                intent.setData(Uri.parse(url));
                startActivity(intent);
            }
        });
    }
}

File: app/src/main/java/com/toosmart/teens/MainActivity.kt
package com.toosmart.teens

import android.content.Intent
import android.net.Uri
import android.os.Bundle
import android.view.View
import android.widget.Button
import androidx.appcompat.app.AppCompatActivity

class MainActivity : AppCompatActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)
        val button1 = findViewById<View>(R.id.page1_button1) as Button
        val button2 = findViewById<View>(R.id.page1_button2) as Button
        val button3 = findViewById<View>(R.id.page1_button3) as Button
        val button4 = findViewById<View>(R.id.page1_button4) as Button
        val button5 = findViewById<View>(R.id.page1_button5) as Button
        val button6 = findViewById<View>(R.id.page1_button6) as Button
        button1.setOnClickListener {
            val url = "https://henricoprevention.org/youth-leadership/"
            val intent = Intent(Intent.ACTION_VIEW)
            intent.setData(Uri.parse(url))
            startActivity(intent)
        }
        button2.setOnClickListener {
            val url = "https://henricoprevention.org/resources/"
            val intent = Intent(Intent.ACTION_VIEW)
            intent.setData(Uri.parse(url))
            startActivity(intent)
        }
        button3.setOnClickListener {
            val url = "https://henricoprevention.org/service-partners/"
            val intent = Intent(Intent.ACTION_VIEW)
            intent.setData(Uri.parse(url))
            startActivity(intent)
        }
        button4.setOnClickListener {
            val url = "https://www.instagram.com/henricotoosmart/"
            val intent = Intent(Intent.ACTION_VIEW)
            intent.setData(Uri.parse(url))
            startActivity(intent)
        }
        button5.setOnClickListener {
            val url = "https://www.youtube.com/channel/UCPoL8cFoxCdPlZFsa8YoLlQ"
            val intent = Intent(Intent.ACTION_VIEW)
            intent.setData(Uri.parse(url))
            startActivity(intent)
        }
        button6.setOnClickListener {
            val url = ""
            val intent = Intent(Intent.ACTION_VIEW)
            intent.setData(Uri.parse(url))
            startActivity(intent)
        }
    }
}